go.mod

```diff
@@ -0,0 +1,60 @@
+module github.com/openshift/cluster-policy-controller
+
+go 1.13
+
+require (
+	github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e
+	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f // indirect
+	github.com/davecgh/go-spew v1.1.1
+	github.com/google/btree v1.0.1-0.20190326150332-20236160a414 // indirect
+	github.com/google/go-cmp v0.4.0 // indirect
+	github.com/gorilla/websocket v1.4.1 // indirect
+	github.com/grpc-ecosystem/go-grpc-middleware v1.1.0 // indirect
+	github.com/jteeuwen/go-bindata v3.0.8-0.20151023091102-a0ff2567cfb7+incompatible
+	github.com/onsi/gomega v1.8.1 // indirect
+	github.com/openshift/api v0.0.0-20200117162508-e7ccdda6ba67
+	github.com/openshift/client-go v0.0.0-20200116152001-92a2713fa240
+	github.com/openshift/library-go v0.0.0-20200120084036-bb27e57e2f2b
+	github.com/prometheus/client_golang v1.1.0
+	github.com/spf13/cobra v0.0.5
+	github.com/spf13/pflag v1.0.5
+	github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5 // indirect
+	go.uber.org/zap v1.13.0 // indirect
+	k8s.io/api v0.17.1
+	k8s.io/apimachinery v0.17.1
+	k8s.io/apiserver v0.17.1
+	k8s.io/client-go v0.17.1
+	k8s.io/component-base v0.17.1
+	k8s.io/klog v1.0.0
+	k8s.io/kubectl v0.0.0
+	k8s.io/kubernetes v1.17.1
+)
+
+replace (
```

Comment:
later try to remove replaces.  My guess is that k/k is requiring several of these replaces.  If so, just comment to that effect.

Reply:
I already tried, the ones required are the top 4, although I might try getting rid of glog now for sure, probably ginkgo too the other two has to stay. The k8s needs to be there since go.mod can't parse them out from k/k since it points to staging :disappointed: